cluster-scope/base/operators.coreos.com/operatorgroups/openshift-cnv/operatorgroup.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: kubevirt-hyperconverged-group
```

Comment:
The name of the parent directory should match the value of name: here in all cases.

Reply:
okay, I'll update the directories. is this the name we are going with then? `kubevirt-hyperconverged-group` with the group suffix? Again, this is all from the redhat documentation and for whatever reason they chose these names.

Reply:
well, if that's the name of the group then it has to be that.